### ISSUE TITLE
Cannot read property 'utp' of null

### ISSUE THREAD

**Issue by proton-ab** [webtorrent/webtorrent#1940]
**What version of this package are you using?**
v0.109.0

**What operating system, Node.js, and npm version?**
```[mimas ~/isla/fetcher] node -v
v12.18.4
[mimas ~/isla/fetcher] yarn -v
1.22.10
[mimas ~/isla/fetcher] uname -a
Linux mimas 4.19.123-feral #2 SMP Mon May 18 09:01:14 BST 2020 x86_64 GNU/Linux
```

**What happened?**
```
/media/md0/eddine/isla/fetcher/node_modules/webtorrent/lib/torrent.js:1733
        const newPeer = this._addPeer(peer.addr, this.client.utp ? 'utp' : 'tcp')
                                                             ^
TypeError: Cannot read property 'utp' of null
    at Timeout._onTimeout (/media/md0/eddine/isla/fetcher/node_modules/webtorrent/lib/torrent.js:1733:62)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
```

**What did you expect to happen?**
No error is thrown

**Are you willing to submit a pull request to fix this bug?**
Not a the moment

**Comment by proton-ab** [comment]
https://github.com/webtorrent/webtorrent/blob/3ece895e8ff5d060da74756267fcc38afb176e37/lib/torrent.js#L1733
`this.client` can be null here because because of race condition (torrent can be destroyed before `_onTimeout` fires)

**Comment by proton-ab** [comment]
This was fixed in https://github.com/webtorrent/webtorrent/pull/1933
What the hell is maintainer of this package doing updating changelog for 0.109.0 but ignoring this bugfix?

**Comment by proton-ab** [comment]
Well, of course it doesn't work. You set `this._reconnectTimeoutId` globally but it's set inside `_drain`; the fix from #1933 would realistically ever work if you had single peer only...

**Comment by feross** [comment]
> What the hell is maintainer of this package doing updating changelog for 0.109.0 but ignoring this bugfix?

Please watch your tone. WebTorrent is a friendly community.

> Well, of course it doesn't work. You define `_reconnectTimeoutId` globally but it's set inside `_drain`; the fix from #1933 would realistically ever work if you had single peer only...

Thanks for noting this. PR sent: https://github.com/webtorrent/webtorrent/pull/1944

**Comment by feross** [comment]
Released as 0.109.2.
